packages/frontend/src/components/Chores/ChoresLayout.js

```diff
@@ -40,41 +51,79 @@ const ChoresLayout = () => {
     }));
   };
 
-  const handleAddChore = () => {
-    const chore = new Chore(
-      newChore.description,
-      newChore.deadline,
-      newChore.points,
-      newChore.assignee
-    );
+//  const handleAddChore = () => {
+//    const chore = new Chore(
+//      newChore.description,
+//      newChore.deadline,
+//      newChore.points,
+//      newChore.assignee
+//    );
+//
+//    setMyHouseholdChores((prevChores) => [
+//      ...prevChores,
+//      chore
+//    ]);
+//
+//    //will have to remove/change this once chores page is completed
+//    // Filter out chores that match the profile name and add them to My Chores
+//    if (
+//      chore.assignee.toLowerCase() ===
+//      "Johnny Clean".toLowerCase()
+//    ) {
+//      setMyChores((prevChores) => [...prevChores, chore]);
+//    }
+//
+//    setNewChore({
+//      description: "",
+//      deadline: new Date(),
+//      points: 0,
+//      assignee: ""
+//    });
+//  };
 
-    setMyHouseholdChores((prevChores) => [
-      ...prevChores,
-      chore
-    ]);
+  const addChore = async () => {
+    try {
+      const deadlineTimestamp = newChore.deadline.getTime();
+      const userIdObject = new mongoose.Types.ObjectId(userId);
+      const householdIdObject = new mongoose.Types.ObjectId(householdId);
 
-    //will have to remove/change this once chores page is completed
-    // Filter out chores that match the profile name and add them to My Chores
-    if (
-      chore.assignee.toLowerCase() ===
-      "Johnny Clean".toLowerCase()
-    ) {
-      setMyChores((prevChores) => [...prevChores, chore]);
-    }
+      const choreToAdd = {
+        chore: newChore.description,
+        completed: false,
+        deadline: deadlineTimestamp,
+        points: newChore.points,
+        householdId: householdIdObject,
+        userId: userIdObject,
+        userName: userName
+      };
 
-    setNewChore({
-      description: "",
-      deadline: new Date(),
-      points: 0,
-      assignee: ""
-    });
-  };
+      const response = await fetch(`${uri}/chore`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          "Authorization": `Bearer ${token}`,
+        },
+        body: JSON.stringify(choreToAdd),
+      });
 
-  //to get token, userid, householdId
-  const token = localStorage.getItem("token");
-  const userId = localStorage.getItem("userId");
-  const householdId = localStorage.getItem("householdId");
+      if (response.ok) {
```

Comment:
Lines 109-112 shows up uber-highlighted for me. Not sure why but I can look into it after class.

Reply:
Hm, same here.. I'll keep a look out as well, thanks for the feedback!